modules/rostests/apitests/acpi/acpi_apitest_whitelist.txt

```diff
@@ -0,0 +1,2 @@
+9bcd83507467 modules/rostests/apitests/acpi/Bus_PDO_QueryResourceRequirements.c 504 "Fails on #WIN2003 #WIN8"
+9bcd83507467 modules/rostests/apitests/acpi/Bus_PDO_QueryResourceRequirements.c 528 "Fails on #WIN2003"
```

Comment:
I'm surprised that the above line includes WIN8 but this one doesn't. The results of this test should not depend on Windows version.

Reply:
Nah, I was just experimenting with stuff.